### ISSUE TITLE
Not generating any output !

### ISSUE THREAD

**Issue by RS-Raghav-Sharma** [glenn2223/vscode-live-sass-compiler#149]
Hello Glenn, 

Thank you for the amazing extension, but it is not working on my new windows laptop. I was using your extension in my previous laptop and it used to work like charm. But now I am having this issue -

![main issue](https://user-images.githubusercontent.com/85699750/149117448-869b8823-7f04-4608-a82d-e6787b6a2df5.png)

![settings](https://user-images.githubusercontent.com/85699750/149117462-40664c9b-c2e1-4671-9eae-d171438bb8ab.png)

The extension is detecting changes but not generating any output whatsoever !

**Comment by glenn2223** [comment]
From your file structure, it looks like you only have a partial on no normal SASS files (one NOT starting with an underscore `_`)

So it can see that you've saved a partial but can't find any none partial files to compile. Try adding a `main.scss` file (or something similarly named) then save `_base.scss` again and it should compile `main.scss`

**Comment by glenn2223** [comment]
>But as I cleared the code in my `main.scss` file, it started giving me this error -

I'm aware and it's fixed in the v5.2-alpha.1 which will be fully released soon. The problem was that there was no CSS to write so it threw an error as such.

> Do I have to keep some code in `main.scss` file ?

It sounds like you need to use the partial file in `main.scss` (as shown below)
```SCSS
@use "base" as *;
```

**Comment by RS-Raghav-Sharma** [comment]
Thank you so much for helping me out Glenn, You are a lifesaver! 

I was stuck with it but because of you, I am finally able to continue my coding.

Thanks a ton 🙌

**Comment by glenn2223** [comment]
Glad I could help
